README.md

```diff
@@ -1,3 +1,19 @@
+======================================================================
+
+# Update
+
+This repo is ancient and largely deprecated.
+
+It largely contains code using proprietary APIs to interface to the VideoCore firmware. We have since moved to standard linux APIs.
+
+Code from here is not installed on latest RPiOS Bookworm images.
```

Comment:
Heads up that we do still ship quite a lot of stuff from this repo.

Reply:
What is still shipped? vcgencmd and dtoverlay? Anything else?
Might be best to move useful stuff into raspberrypi/utils.

Reply:
[full list here](https://github.com/raspberrypi/userland/files/13043417/files.txt)